crates/core/tedge_api/src/health.rs

```diff
@@ -44,7 +44,7 @@ impl ServiceHealthTopic {
         }
     }
 
-    pub fn up_message(&self) -> Message {
+    pub fn up_message(&self, mqtt_schema: &MqttSchema) -> Message {
```

Comment:
Makes me wonder why the schema is not part of the state of the `ServiceTopic` so that this `up_message` is generated using the same schema that was used to create it.

Reply:
This needs more changes and a bit of refactoring. So, its better to do it as part of a followup PR.

Reply:
Done with https://github.com/thin-edge/thin-edge.io/pull/2386/commits/deb7e20ae253edf248e772133f00612206dfe605